infra/aws/terraform/prow-build-cluster/hack/flux-update.bash

```diff
@@ -65,6 +65,20 @@ flux create hr node-termination-handler \
     --interval=${sync_interval} \
     --export >> ${resources_dir}/kube-system/flux-hr-node-termination-handler.yaml
 
+boilerplate > ${resources_dir}/flux-system/flux-source-helm-kubecost-chart.yaml
+flux create source helm kubecost \
+    --url https://kubecost.github.io/cost-analyzer \
+    --interval=${sync_interval} \
+    --export >> ${resources_dir}/flux-system/flux-source-helm-kubecost-chart.yaml
+
+boilerplate > ${resources_dir}/kubecost/flux-hr-kubecost.yaml
+flux create helmrelease kubecost --chart cost-analyzer \
+  --source HelmRepository/kubecost \
+  --chart-version 1.107.1 \
+  --namespace kubecost \
+  --values-from=ConfigMap/kubecost-values \
+  --export >> ${resources_dir}/kubecost/flux-hr-kubecost.yaml
```

Comment:
Some suggestions to make the command consistent with others

```suggestion
flux create hr kubecost \
    --source=HelmRepository/kubecost.flux-system \
    --namespace=kubecost \
    --chart=cost-analyzer \
    --chart-version=1.107.1 \
    --values-from=ConfigMap/kubecost-values \
    --interval=${sync_interval} \
    --export >> ${resources_dir}/kubecost/flux-hr-kubecost.yaml
```

Reply:
Maybe we should check if we can explicitly specify the namespace for `kubecost-values` using `.<namespace>` as for `--source`

Reply:
Flux documentation does not mention about the namespace:

```bash
--values-from strings   a Kubernetes object reference that contains the values.yaml data key in the format '<kind>/<name>', where kind must be one of: (Secret,ConfigMap)

```

Reply:
Okay, then we're all good here 👍